### ISSUE TITLE
Is this still going?

### ISSUE THREAD

**Issue by fuki98** [lrre-foss/rnr#117]
.

**Comment by sorket** [comment]
Yes, RNR is still going. We have restarted development (hence why the repository can look rather empty compared to our videos) and it has stalled due to developers becoming busy with other projects and/or real life situations. It's not over, we're just busy with other things at the moment.

**Comment by fuki98** [comment]
Awesome
